src/SkynetServer/Network/Model/ChannelAction.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SkynetServer.Network.Model
+{
+    internal enum ChannelAction
+    {
+        None,
+        Typing,
+        RecordingAudio
```

Comment:
Should we add something like `TakingPicture` here? I find it annoying when I ask someone on WhatsApp to send an image of something, and then they don't read my messages and I don't know whether they are in the `CameraActivity` or just plain ignoring me.

Reply:
When I'm supposed to take and send multiple pictures to anyone, I usually use the Camera app and share these pictures afterwards. If we want to have something reliable we would have to implement a feature to share your foreground app like Discord does for games. But then you should open an issue in skynet-im/skynet.

Reply:
alright